### ISSUE TITLE
Contrast ratio too low

### ISSUE THREAD

**Issue by alexandregaliay** [alexandregaliay/alexandregaliay-com#2]
According to Lighthouse, the contrast ratio is too low on orange links.

>> Switch background color to #23272E (Raisin Black on coolors.co)
